grammars/javascript.cson

```diff
@@ -1135,6 +1134,36 @@
         ]
       }
     ]
+  'function_methods':
```

Comment:
Would it be better to rename this `method_functions` (or maybe even `methods`), and the one below it just `functions`?  I think that clears up some of the ambiguity between `function_call` and `function_calls`.